cloud-sql/mysql/mysql/package.json

```diff
@@ -10,51 +9,25 @@
     "url": "https://github.com/GoogleCloudPlatform/nodejs-docs-samples.git"
   },
   "engines": {
-    "node": ">=8.0.0"
+    "node": ">=10.0.0"
   },
   "scripts": {
     "system-test": "mocha test/*.test.js --timeout=60000 --exit",
     "test": "npm run system-test"
   },
   "dependencies": {
     "@google-cloud/logging-winston": "^3.0.0",
-    "body-parser": "1.19.0",
+    "body-parser": "^1.19.0",
     "express": "^4.17.1",
     "promise-mysql": "^4.0.0",
     "prompt": "^1.0.0",
     "pug": "^2.0.3",
     "winston": "^3.1.0"
   },
   "devDependencies": {
-    "@google-cloud/nodejs-repo-tools": "^3.3.0",
     "mocha": "^6.0.0",
     "proxyquire": "^2.1.0",
     "supertest": "^4.0.0",
     "sinon": "^7.1.1"
-  },
-  "cloud-repo-tools": {
-    "requiresKeyFile": true,
-    "requiresProjectId": true,
-    "test": {
-      "app": {
-        "requiredEnvVars": [
```

Comment:
Any context on why this is being removed? I thought this was what throws errors if the variables are missing, which seems useful for folks using the samples.

Reply:
I think the sample rubrics specifically call out that we're heavily optimizing for the cloud-site embedding scenario for these samples over the idea of folks just cloning and running the repositories.  These would only apply in the case of running the tests, right?

Reply:
We're removing `nodejs-repo-tools`. These checks are specific to that library.